frontend/src/scenes/notebooks/Nodes/NotebookNodeExperiment.tsx

```diff
@@ -0,0 +1,145 @@
+import { LemonButton, LemonDivider } from '@posthog/lemon-ui'
+import { urls } from 'scenes/urls'
+import { LemonSkeleton } from 'lib/lemon-ui/LemonSkeleton'
+import { notebookNodeLogic } from './notebookNodeLogic'
+import { NotebookNodeViewProps } from '../Notebook/utils'
+import { ExperimentLogicProps, experimentLogic } from 'scenes/experiments/experimentLogic'
+import { buildFlagContent } from './NotebookNodeFlag'
+import { useEffect } from 'react'
+import { ExperimentPreview } from 'scenes/experiments/ExperimentPreview'
+import { insightLogic } from 'scenes/insights/insightLogic'
+import { EXPERIMENT_INSIGHT_ID } from 'scenes/experiments/constants'
+import { funnelDataLogic } from 'scenes/funnels/funnelDataLogic'
+import { trendsDataLogic } from 'scenes/trends/trendsDataLogic'
+import { ExperimentResult } from 'scenes/experiments/ExperimentResult'
+
+const Component = (props: NotebookNodeViewProps<NotebookNodeExperimentAttributes>): JSX.Element => {
+    const { id } = props.node.attrs
+    const { experiment, experimentLoading, statusTag, resultsTag, isExperimentRunning } = useValues(
+        experimentLogic({ experimentId: id })
+    )
+    const { loadExperiment } = useActions(experimentLogic({ experimentId: id }))
+    const { expanded } = useValues(notebookNodeLogic)
+    const { insertAfter } = useActions(notebookNodeLogic)
+
+    // experiment progress details
+    const logic = insightLogic({ dashboardItemId: EXPERIMENT_INSIGHT_ID })
+    const { insightProps } = useValues(logic)
+
+    const { conversionMetrics, results } = useValues(funnelDataLogic(insightProps))
+    const { results: trendResults } = useValues(trendsDataLogic(insightProps))
+
+    const conversionRate = conversionMetrics.totalRate * 100
+    const trendCount = trendResults[0]?.count
+    const entrants = results?.[0]?.count
+
+    useEffect(() => {
+        loadExperiment()
+    }, [id])
+
+    return (
+        <div>
+            <BindLogic logic={experimentLogic} props={{ experimentId: id }}>
+                <div className="flex items-center gap-2 p-3">
+                    <IconExperiment className="text-lg" />
+                    {experimentLoading ? (
+                        <LemonSkeleton className="h-6 flex-1" />
+                    ) : (
+                        <>
+                            <span className="flex-1 font-semibold truncate">{experiment.name}</span>
+                            {statusTag}
+                            {resultsTag}
+                        </>
+                    )}
+                </div>
+
+                {expanded ? (
+                    <>
+                        {experiment.description && (
+                            <>
+                                <LemonDivider className="my-0" />
+                                <span className="p-2">{experiment.description}</span>
+                            </>
+                        )}
+                        {id && !experiment.start_date && (
```

Comment:
How would someone end up in a scenario where the id is not defined? Would that not mean that there was no associated experiment